installing-tas-for-kubernetes.html.md.erb

```diff
@@ -79,23 +79,16 @@ To install TAS for Kubernetes:
 1. Change directory to the `tanzu-applications-service` directory.
 
 1. To deploy TAS for Kubernetes using the images in the private registry:
-    * If your client has the private registry CA certificates:  
```

Comment:
The kbld command does not connect to registry, so there is no meaning to add certificate ignore flags